src/transformers/models/auto/processing_auto.py

```diff
@@ -57,6 +57,7 @@
         ("layoutlmv2", "LayoutLMv2Processor"),
         ("layoutlmv3", "LayoutLMv3Processor"),
         ("markuplm", "MarkupLMProcessor"),
+        ("mctct", "MCTCTProcessor"),
```

Comment:
Surprised this wasn't caught actually by the doctest: https://huggingface.co/docs/transformers/model_doc/mctct#transformers.MCTCTModel.forward.example

Reply:
Yeah I was trying a variant of this code  fragment when I noticed this.

Do the doc tests actually run the code fragments and check for errors?

Reply:
The MCTCT model is not among the files tested for doctests.